app/controllers/admin/activities_controller.rb

```diff
@@ -1,11 +1,12 @@
 #:nodoc:
 class Admin::ActivitiesController < ApplicationController
+  include ::StudyYear
   # replaced with calls in each of the methods
   # impressionist :actions => [ :create, :update, :destroy ]
 
   def index
     @activities = Activity.study_year(params['year']).order(start_date: :desc)
-    @years = (Activity.take(1).first.start_date.year..Date.today.study_year).map { |year| ["#{ year }-#{ year + 1 }", year] }.reverse
+    @years = range(Activity.first.start_date.year).reverse
```

Comment:
It takes the first activity from the database and uses its `start_date`. I tried to keep it simple, this will work in most cases since the first activity is long ago